upp/main.py

```diff
@@ -25,7 +25,7 @@ class HelpFormatter(argparse.RawTextHelpFormatter, argparse.ArgumentDefaultsHelp
 
 
 def parse_args():
-    abool = argparse.BooleanOptionalAction
+    abool = "store_true"
```

Comment:
I don't think this new behaviour will be consistent with the old - `BooleanOptionalAction` creates the corresponding negative args so that stages can be disabled. I think you will have ot add these manually now - take a look [here](https://stackoverflow.com/questions/52403065/argparse-optional-boolean)

Reply:
Ok. I will add the negative flags

Reply:
I have done it now with the most rigorous testing possible